GPU/Common/FramebufferCommon.cpp

```diff
@@ -90,6 +90,10 @@ void CenterDisplayOutputRect(float *x, float *y, float *w, float *h, float origW
 		float origRatio = !rotated ? origW / origH : origH / origW;
 		float frameRatio = frameW / frameH;
 		
+		// assume aspect ratios are the same when they are close enough.
```

Comment:
What was this here to fix?

-[Unknown]

Reply:
a 480x272 psp frame on a 854x480 framebuffer (Wii-U gamepad resolution)  would generate coordinates that don't fill the entire viewport.
I wrote this while trying to fix a display issue with the software renderer, but it didn't help much. maybe the difference is actually smaller than a pixel to even matter, I'll test it again.